modules/plot/main/coffee/Graph.coffee

```diff
@@ -35,7 +35,9 @@ class Graph extends hx.EventEmitter
 
     selection = hx.select(@selector)
 
-    if @_.options.redrawOnResize then selection.on 'resize', 'hx.plot', => @render()
+    selection.on 'resize', 'hx.plot', =>
+      @render() if @_.options.redrawOnResize
```

Comment:
This fixes the issue where setting up graph with `redrawOnResize` and then changing the value using the setter/getter doesn't actually _do_ anything because the 'resize' event is created in the constructor